commands/command_utils.py

```diff
@@ -48,7 +49,8 @@ def detect_sap_hana():
     if os.path.exists(HANA_BASE_PATH):
         for entry in os.listdir(HANA_BASE_PATH):
             # Does /hana/shared/{entry}/exe/linuxx86_64/hdb/sapcontrol exist?
-            if os.path.exists(os.path.join(HANA_BASE_PATH, entry, HANA_SAPCONTROL_PATH)):
+            if os.path.exists(os.path.join(HANA_BASE_PATH, entry, HANA_SAPCONTROL_PATH_X86_64)) or \
+               os.path.exists(os.path.join(HANA_BASE_PATH, entry, HANA_SAPCONTROL_PATH_PPC64LE)):
```

Comment:
Do not break lines explicitly, instead:
```suggestion
            if (os.path.exists(os.path.join(HANA_BASE_PATH, entry, HANA_SAPCONTROL_PATH_X86_64))
               or os.path.exists(os.path.join(HANA_BASE_PATH, entry, HANA_SAPCONTROL_PATH_PPC64LE))):
```
the same on other places in the code